samples/trumpify/trumpify.py

```diff
@@ -0,0 +1,120 @@
+import os
+import sys
+import threading
+import time
+
+sys.path.append(os.path.abspath('../../'))
+
+from samples.trumpify.utils import TwitterUtils
+from daemo.client import DaemoClient
+
+CREDENTIALS_FILE = 'credentials.json'
+
+PROJECT_ID = -1
+
+INPUT_TWITTER_NAME = 'HillaryClinton'
+MINUTES = 60
+FETCH_INTERVAL_MIN = 5
+TWEET_COUNT = 10
+MONITOR_INTERVAL_MIN = 60
+
+twitter = TwitterUtils()
+daemo = DaemoClient(CREDENTIALS_FILE)
+
+
+def transform_new_tweets(twitter_name, count, interval):
+    """
+    Fetches "count" number of tweets from twitter feed of "twitter_name" after every "interval" seconds and posts them
+    to Daemo server for translation
+
+    :param twitter_name: twitter account to fetch tweets from
+    :param count: number of tweets to fetch
+    :param interval: period in seconds to repeat the process
+    """
+    while True:
+        messages = twitter.fetch_tweets(twitter_name=twitter_name, count=count, interval=interval)
+
+        if len(messages) > 0:
```

Comment:
This check is there only to have that print statement when there is no tweets in last 5 min.